locale/de.properties

```diff
@@ -0,0 +1,69 @@
+frequently_asked_questions = Häufig gestellte Fragen (FAQ)
+from_these_sites = auf dieser Seite. Mit den Schiebern lässt sich einstellen wie der Privacy Badger mit den einzelnen Trackern umgehen soll.
+so_far = bis jetzt. Mit den Schiebern kannst Du kontrollieren wie der Privacy Badger mit den einzelnen Trackern umgehen soll.
+loading = Lade...
+notracking = Die folgenden Seiten scheinen dich nicht zu tracken
+potential = Potenzial
+privacy_badger = Privacy Badger
+first_run_text = Erfahre wie der Privacy Badger deine Privatsphäre schützt.
+report_bug = Melde fehlerhafte Webseite
+report_title = Melde einen Fehler
+report_text = Bitte melde deinen Fehler hier
+report_terms = Folgende Informationen werden automatisch an die EFF gesendet: Die Seite die Du aktuell besuchst, deine Browser Version, die Version des Privacy Badgers und der Status der einzelnen Schieber für diese Seite.
+report_button = Abschicken!
+report_cancel = Abbrechen
+report_input_label = Beschreibung
+report_field = Was scheint das Problem zu sein?
+report_success = Danke! Wir werden den auf den Grund gehen.
+report_fail = Oops. Irgendwas ist schief gegangen.
+report_close = Schließen
+restore_button = Das wird <b>alle</b> Tracker Einstellungen zurücksetzen (grün wenn Du Drittanbieter Cookies im Firefox aktiviert hast, ansonsten gelb). Willst Du wirklich fortfahren?
+settings_disable = Deaktivieren auf derzeitiger Seite
+settings_report  = Einen Fehler melden. . .
+settings_unblock = Entsperren aller Tracker. . .
+status_none_detected = Es sind keine Dritt-Anbieter Quellen auf dieser Seite.
+status_reload = Seite aktualisieren um aktive Tracker zu sehen.
+trackers = Tracker
+tracking_domains = Tracking Adressen
+unblock_all = Alle Tracker entsperren. . .
+options_title = Privacy Badger Einstellungen
+options = Einstellungen
+whitelisted_domains = Erlaubte Adressen (Whitelist)
+options_filter_settings = Filter Einstellungen
+options_misc_prefs = Andere Einstellungen
+disabled_for_these_domains = Der Privacy Badger ist auf folgenden Webseiten deaktiviert
+add_domain_button = Webseite hinzufügen
+remove_button = Webseite entfernen
+options_slider_text = Bewege die Schieber um Webseiten zu sperren/entsperren
+options_loading = Lade...
+options_social_enable = Aktiviert das Blocken von Social Widgets
+firstrun_by_eff = Ein Projekt der Electronic Frontier Foundation
+firstrun_title = Danke das Du den Privacy Badger installiert hast!
+firstrun_social_share = Lass die Welt wissen das Du dich gegen Online Tracking wehrst.
+slideshow_1 = Hallo, ich bin der Privacy Badger! Danke das Du mich installiert hast. Damit wir gut zusammenarbeiten können, möchte ich dir ein paar hilfreiche Tipps geben.
+slideshow_2 = Ich werde hier oben in deinen Browser nach Werbung und Trackern Ausschau halten, die verfolgt was du im Internet machst.
+slideshow_3 = Wenn ich feststelle das etwas versucht dich zu verfolgen verhindere ich das.  
```

Comment:
Yes it sounds a bit weird in both ways, but as the text seems to be a "personal" introduction of the badger I think it is okay to say "the".